cmd/http.go

```diff
@@ -89,14 +89,20 @@ Examples:
   # HTTP HEAD request to jsdelivr.com from 2 probes in New York (protocol, port and path are inferred from the URL)
   http https://www.jsdelivr.com:443/package/npm/test?nav=stats from New York --limit 2
 
-  # HTTP GET request to google.com from 2 probes from London or Belgium
-  http google.com from London,Belgium --limit 2 --method get
+  # HTTP GET request to google.com from 2 probes from London or Belgium in CI mode
+  http google.com from London,Belgium --limit 2 --method get --ci
 
   # HTTP HEAD request to jsdelivr.com from a probe that is from the AWS network and is located in Montreal using HTTP2
   http jsdelivr.com from aws+montreal --protocol http2
```

Comment:
Maybe also add an example of custom HTTP headers?

Reply:
@jimaek headers are not implemented in the CLI at the moment. I can see a couple of spots in the code where the comment `// TODO: Headers` was added

Reply:
https://github.com/jsdelivr/globalping-cli/issues/44